docs/text_handling.md

```diff
@@ -4,6 +4,19 @@ OpenVINO Model Server can now greatly simplify writing the applications with Nat
 
 We addressed both the situation when the original model requires tokens on input or output and there is added support for models with embedded tokenization layer. Below are demonstrated use cases with a simple client application sending and receiving text in a string format. Whole complexity of the text conversion is fully delegated to the remote serving endpoint.
 
+## Accepting strings by serving Python script
```

Comment:
```suggestion
## Serving a pipeline with tokenization and detokenization delegated to a python node
```

Reply:
I don't think it is a good title, since strings can be used for other things than tokenization and detokenization when serving python script. @dtrawins